python/tk_desktop/desktop_window.py

```diff
@@ -71,6 +72,20 @@ def __init__(self, parent=None):
         self.__pipeline_configuration_separator = None
         self._settings_manager = settings.UserSettings(sgtk.platform.current_bundle())
 
+        engine = sgtk.platform.current_engine()
+
+        self._toolkit_manager = ToolkitManager(engine.get_current_user())
+        self._toolkit_manager.caching_policy = ToolkitManager.CACHE_FULL
+        self._toolkit_manager.plugin_id = "config.basic"
```

Comment:
Should it? It's the configuration for the project, not the desktop. Surely you don't want a project specific config to use the same plugin id as the one for the desktop?